sources/packages/backend/workflow/test/2021-2022/parttime-assessment/awards-amounts/parttime-assessment-2021-2022-awards-amount-CSGD.e2e-spec.ts

```diff
@@ -122,6 +122,16 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-awards-amount-CS
           .limitAwardCSGDAmount,
       ),
     );
+    expect(calculatedAssessment.variables.federalAwardNetCSGDAmount).toBe(
```

Comment:
In the previous PRs we talked about using raw values instead of formulas in expect.